tests/microsoft/azure/sensors/test_wasb.py

```diff
@@ -25,7 +27,15 @@ class TestWasbBlobSensorAsync:
         blob_name=TEST_DATA_STORAGE_BLOB_NAME,
     )
 
-    def test_wasb_blob_sensor_async(self):
+    @mock.patch(f"{MODULE}.WasbBlobSensorAsync.defer")
+    @mock.patch(f"{MODULE}.WasbBlobSensorAsync.poke", return_value=True)
+    def test_WasbBlobSensorAsync_async_finish_before_deferred(self, mock_poke, mock_defer, context):
```

Comment:
Can we use a snake case here for the test name? also async is repeated twice in the name.

Reply:
ah yes, I forget to update it. let me fix it for all my branches. Thanks!

Reply:
Just fixed it